proposals/0000-reduce-fragmentation.md

```diff
@@ -0,0 +1,332 @@
+
+```jsx
+<TextArea
+  autoComplete="email"
+  inputMode="email"
+/>
+
+<TextArea
+  multiline
+  readOnly
+  rows={3}
+/>
+```
+
+### New styles
+
+* CSS Custom Properties
+  * Support CSS custom property syntax `--variable-name`.
+  * This can be shimmed in user-space on top of the existing `StyleSheet` API, with a React Context used to provide variables and values to a subtree.
+* CSS Lengths
+  * Support `em`, `rem`, `v*`, `px` length units.
+  * This can be shimmed in user-space on top of the existing `StyleSheet` API, with a React Context used to provide ancestral font-size information for `rem` values.
+* CSS Colors
+  * Support CSS Colors using [Colorjs.io](https://colorjs.io/).
+  * This can be shimmed in user-space.
+* CSS Media Queries
+  * Support CSS Media Queries. Although Media Queries are not a preferred long-term solution, [Container Queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries) are not yet widely supported by browsers.
+  * This can be shimmed in user-space on top of the existing `StyleSheet` API.
+* CSS Logical Properties
+  * Support CSS logical property names, i.e., `paddingInlineStart` rather than only non-standard `paddingStart`.
+  * Support subtree-level logical properties, where setting the `dir` prop to `ltr` or `rtl` alters the way logical properties are resolved. Currently, React Native only supports global writing direction configuration for Android.
+* `aspectRatio`
+  * Support string values, i.e., `'16 / 9'`, to align with CSS.
+* `backgroundImage`
+  * Support setting background images via `url()`.
+  * Optional: support CSS gradients.
+* `boxShadow`
+  * Add native support for CSS box shadows.
+  * This would replace the very buggy iOS-specific `shadow*` styles and Android `elevation` style, allowing developers to unify shadows across native and web platforms.
+* `fontVariant`
+  * Support space-separated string values to align with CSS.
+* `fontWeight`
+  * Support number values to align with React DOM / CSS.
+* `lineClamp`
+  * Support CSS clamping text to a set number of lines via style.
+  * This would be equivalent to using the `numberOfLines` prop on `<Text>` components.
+* `objectFit`
+  * Support CSS `objectFit` property for `<Image>`.
+  * This would be a subset (no "repeat" keyword) of functionality currently supported by the `resizeMode` prop and style for `<Image>`.
+  * `objectFit === 'contain'` => `resizeMode = 'contain'`
+  * `objectFit === 'cover'` => `resizeMode = 'cover'`
+  * `objectFit === 'fill'` => `resizeMode = 'stretch'`
+  * `objectFit === 'scale-down'` => `resizeMode = 'contain'`
+* `pointerEvents`
+  * Support setting the `pointerEvents` values via style rather than a prop.
+  * Retain the React Native specific `box-none` and `box-only` values.
+* `textShadow`
+  * Add native support for CSS text shadows.
+* `transform`
+  * Support using string values to set transforms.
+  * This can be shimmed in user-space with a parser.
+* `verticalAlign`
```

Comment:
I wouldn't be heartbroken if `verticalAlign` is deprioritized/never added. It's up there next to `float` in terms of unintuitive CSS APIs.